src/webapp/helpers/mapbox-coordinates.js

```diff
@@ -0,0 +1,18 @@
+import { LngLat } from "mapbox-gl";
```

Comment:
i should really test this, will TODO. for now they are simple enough that i am not too worried but yeah.